server/controllers/stock/core.js

```diff
@@ -233,6 +234,100 @@ function getLots(sqlQuery, parameters, finalClause = '', orderBy = '') {
   return db.exec(query, queryParameters);
 }
 
+/**
+ * Get asset info
+ *
+ * @param {string} depotUuid
+ * @param {object} params
+ */
+async function getAssets(params, finalClause) {
+
+  // NOTE: Expected parameters
+  //   depot_uuid, is_asset, scan_start_date, scan_end_date, reference_number
+  //   scan_status ==> one of 'all', 'scanned', or 'unscanned'
+
+  // Construct the WHERE clause to insert inside the join
+  // (Not obvious how to do this with filters since it is inside a JOIN)
+  let scanPeriod = '';
+  if ('scan_start_date' in params) {
+    const startDate = moment(params.scan_start_date).format('YYYY-MM-DD HH:MM:SS');
+    scanPeriod = `scan.created_at >= DATE('${startDate}')`;
+  }
+  if ('scan_end_date' in params) {
+    if (scanPeriod) {
+      scanPeriod += ' AND ';
+    }
+    const endDate = moment(params.scan_end_date).format('YYYY-MM-DD HH:MM:SS');
+    scanPeriod += `scan.created_at <= DATE('${endDate}')`;
+  }
+  if (scanPeriod) {
+    scanPeriod = `WHERE ${scanPeriod}`;
+  }
+
+  const sql = `
+    SELECT BUID(l.uuid) AS uuid, l.label, l.description AS lot_description,
```

Comment:
This query is pretty large without much reason to be so that I can tell.

Some easy wins: 
  - Remove references to `tags` and `lot_tags`. 
  - Remove columns/references to expiration dates.
  - Remove references to `stock_value`.

Reply:
Made these mods
- Removed tags
- Removed refs to stock_value

Did NOT remove expiration date.   Currently we are allowing scanning to apply to assets or other stock.    To remove some of these things, we need to decide if Assets scans apply ONLY to assets.  If so, some changes will be needed.  Given that future projects may want to track assets and regular interchangeable stock (eg, scalpels), we may need to clean up some of the labeling.